memote/suite/tests/test_sbo.py

```diff
@@ -261,8 +263,11 @@ def test_sink_specific_sbo_presence(read_only_model):
     ann = test_sink_specific_sbo_presence.annotation
     ann["data"] = get_ids(sbo.check_component_for_specific_sbo_term(
         helpers.find_sink_reactions(read_only_model), "SBO:0000632"))
-    ann["metric"] = len(ann["data"]) / len(
-        helpers.find_sink_reactions(read_only_model))
+    try:
+        ann["metric"] = len(ann["data"]) / len(
+            helpers.find_sink_reactions(read_only_model))
```

Comment:
Since it's called multiple times (see above) can you please store the result of `find_sink_reactions` in a local variable.